src/firestore/indexes.ts

```diff
@@ -395,7 +416,7 @@ export class FirestoreIndexes {
     const url = field.name;
```

Comment:
nit: The comments for this method should be updated(on line 413; cannot directly comment on that line for some reason)